### PR TITLE
[TM] Chp4-2 First steps of labeling using a point layer

### PR DESCRIPTION
**Pull request by DelazJ**
Instead of using the label toolbar we move to the label tab in layer properties
(easier as a default tool and more featured). The toolbar should be kept for advanced steps.

### CODE REVIEW

File: source/docs/training_manual/vector_classification/label_tool.rst [hunk 1]
```diff
@@ -5,104 +5,122 @@
...
+In order to display textual information about a layer using its attributes:
 
-* Go to the menu item :menuselection:`View --> Toolbars`.
-* Ensure that the :guilabel:`Label` item has a check mark next to it. If it
-  doesn't, click on the :guilabel:`Label` item, and it will be activated.
-* Click on the :guilabel:`places` layer in the :guilabel:`Layers` panel, so that
-  it is highlighted.
-* Click on the following toolbar button: |labeling|
+#. Select the :guilabel:`places` layer and press the |symbology| :sup:`Open
+   Layer Styling panel` button. You can as well use the :guilabel:`Layer
```
Comment (havatv) [inline source/docs/training_manual/vector_classification/label_tool.rst]:
```suggestion
   Layer Styling panel` button. You can also use the :guilabel:`Layer
```

File: source/docs/training_manual/vector_classification/label_tool.rst [hunk 2]
```diff
@@ -5,104 +5,122 @@
...
+      :align: center
 
-* Check the box next to :guilabel:`Label this layer with...`.
+   |
 
-You'll need to choose which field in the attributes will be used for the
-labels. In the previous lesson, you decided that the :kbd:`NAME` field was the
-most suitable one for this purpose.
+   The dialog is filled with multiple options.
+   You'll need to choose which field in the attributes will be used for the
```
Comment (havatv) [inline source/docs/training_manual/vector_classification/label_tool.rst]:
```suggestion
   You will need to choose the field /attribute to use for labelling.
```

File: source/docs/training_manual/vector_classification/label_tool.rst [hunk 3]
```diff
@@ -5,104 +5,122 @@
...
+   labels. In the previous lesson, you decided that the :guilabel:`name` field
```
Comment (havatv) [inline source/docs/training_manual/vector_classification/label_tool.rst]:
```suggestion
   In the previous lesson, you decided that the :guilabel:`name` field
```

File: source/docs/training_manual/vector_classification/label_tool.rst [hunk 4]
```diff
@@ -5,104 +5,122 @@
...
 
 |basic| |FA| Changing Label Options
 -------------------------------------------------------------------------------
 
-Depending on the styles you chose for your map in earlier lessons, you'll
+Depending on the styles you chose for your map in earlier lessons, you
 might find that the labels are not appropriately formatted and either overlap or
-are too far away from their point markers.
+are too far away from their point markers. Or you may simply not like the
+default rendering. Let's explore some of the label options to repair that.
```
Comment (havatv) [inline source/docs/training_manual/vector_classification/label_tool.rst]:
```suggestion
default rendering. Let's explore some of the label options to fix that.
```

File: source/docs/training_manual/vector_classification/label_tool.rst [hunk 5]
```diff
@@ -5,104 +5,122 @@
...
 
-* Open the :guilabel:`Label tool` again by clicking on its button as before.
-* Make sure :guilabel:`Text` is selected in the left-hand options list, then
-  update the text formatting options to match those shown here:
+#. Reopen the |labeling| :guilabel:`Labels` tab.
+#. Select the |labelbuffer| :sup:`Buffer` tab in the upper-hand  options list
+   (or left-hand if you are in the :guilabel:`Layer Properties` panel - in this
+   case remember to click :guilabel:`Apply` to see your changes on the map).
+#. Tick the checkbox of the |unchecked| :guilabel:`Draw text buffer` option,
+   then set a buffer :guilabel:`Size` of ``0.8 mm``:
```
Comment (havatv) [inline source/docs/training_manual/vector_classification/label_tool.rst]:
```suggestion
   then set a buffer :guilabel:`Size` to ``0.8 mm``:
```

File: source/docs/training_manual/vector_classification/label_tool.rst [hunk 6]
```diff
@@ -5,104 +5,122 @@
...
+   .. image:: img/buffer_results.png
+      :align: center
+      :scale: 60
 
-.. image:: img/buffer_options.png
-   :align: center
+   |
 
-* Click :guilabel:`Apply`.
+   However, the labels appear too large next to the other features on the map.
```
Comment (havatv) [inline source/docs/training_manual/vector_classification/label_tool.rst]:
```suggestion
   Now the labels appear too large next to the other features on the map.
```

File: source/docs/training_manual/vector_classification/label_tool.rst [hunk 7]
```diff
@@ -5,104 +5,122 @@
...
+#. Enable the |text| :guilabel:`Text` tab and set a lower :guilabel:`Size`, e.g.
```
Comment (havatv) [inline source/docs/training_manual/vector_classification/label_tool.rst]:
```suggestion
#. Enable the |text| :guilabel:`Text` tab and set a smaller :guilabel:`Size`, e.g.
```

File: source/docs/training_manual/vector_classification/label_tool.rst [hunk 8]
```diff
@@ -5,104 +5,122 @@
...
+   ``8 points``.
 
+   Better but these long labels are still not good looking.
```
Comment (havatv) [inline source/docs/training_manual/vector_classification/label_tool.rst]:
```suggestion
   Better, but these long labels are still not good looking.
```

File: source/docs/training_manual/vector_classification/label_tool.rst [hunk 9]
```diff
@@ -5,104 +5,122 @@
...
+   :scale: 60
 
-* Click :guilabel:`Apply`.
-
-
-You'll see that the labels are no longer overlapping their point markers.
-
+.. Todo: Use DD button to constrain the label to the right of the marker
+ instead of the default top right or let it fo the DD settings section (would
+ require to save the style - not taught yet!)
```
Comment (havatv) [inline source/docs/training_manual/vector_classification/label_tool.rst]: Is this necessary? Where is the DD button?

File: source/docs/training_manual/vector_classification/label_tool.rst [hunk 10]
```diff
@@ -124,28 +142,26 @@ the point symbols and replace them completely with labels.
...
+#. Open the :guilabel:`Layer labeling settings` dialog for the
+   :guilabel:`places` layer
+#. Select the |labelplacement| :guilabel:`Placement` option from the options list.
+#. Click on the :guilabel:`Offset from point` button
 
-This will reveal the :guilabel:`Quadrant` options which you can use to set the
-position of the label in relation to the point marker. In this case, we want the
-label to be centered on the point, so choose the center quadrant:
+   This will reveal the :guilabel:`Quadrant` options which you can use to set
+   the position of the label in relation to the point marker. In this case, we
```
Comment (havatv) [inline source/docs/training_manual/vector_classification/label_tool.rst]:
```suggestion
   the position of the label relative to the point marker. In this case, we
```

File: source/docs/training_manual/vector_classification/label_tool.rst (hunk 9 above)
Comment (havatv) [reply source/docs/training_manual/vector_classification/label_tool.rst]: OK.

**Comment by havatv** [comment]
Any reason for not moving forward on this one, @DelazJ ?

**Comment by havatv** [comment]
Could you have a look at this one, @DelazJ?  It has been hanging around for quite a long time now.

**Comment by DelazJ** [comment]
This is an unfinished work I wrongly pushed as I mentioned in a previous comment. Update/complete this to a mergeable state requires me to go through the exercises and I currently don't have time to spend on that so I'm closing for the moment, to "declutter" the queue. Will see if/when i self-assign less other "priorities" and have more time...

**Pull request closed without merging**